packages/wasm-dpp/src/lib.rs

```diff
@@ -14,11 +14,14 @@ pub mod errors;
 mod identifier;
 mod identity;
 mod metadata;
+pub mod mocks;
```

Comment:
Shouldn't mocks be behind a feature flag?

Reply:
wow, something strange happened here. Maybe because merging. The `mocks` shoudldn't exist at all. And I moved `DocumentsTransitionWasm` which was in `mocks.rs` for some reason :D